server/channelserver/handlers_quest.go

```diff
@@ -195,23 +196,50 @@ func handleMsgMhfEnumerateQuest(s *Session, p mhfpacket.MHFPacket) {
 	bf := byteframe.NewByteFrame()
 	bf.WriteUint16(0)
 
-	rows, _ := s.server.db.Query("SELECT id, COALESCE(max_players, 4) AS max_players, quest_type, quest_id, COALESCE(mark, 0) AS mark FROM event_quests ORDER BY quest_id")
+	currentTime := time.Now()
+	var tableName = "event_quests"
+	/* This is an example of how I have to test and avoid issues, might be a thing later?
+ 	if _config.ErupeConfig.RealClientMode <= _config.F5 {
+		tableName = "event_quests_older"
+	}
+ 	*/
+	// Check the event_quests table to load the quests with rotation system
+	rows, err := s.server.db.Query("SELECT id, COALESCE(max_players, 4) AS max_players, quest_type, quest_id, COALESCE(mark, 0) AS mark, start_time, active_duration, inactive_duration FROM " + tableName + "")
```

Comment:
If you were to keep the tableName variable rather than hardcoding it, I think it'd be better to use string.Format here.